src/atoms/RoundedButtons.js

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import styled, { css, withTheme } from 'styled-components';
+
+import { getColor, getVariable } from '../utils';
+
+const StyledLink = styled.a`
+  font-family: ${getVariable('mainFont')};
+  font-weight: ${getVariable('semiBoldWeight')};
+  display: inline-flex;
+  align-items: center;
+  border: 0 none;
+  background: ${props => (props.background ? getColor(props.background) : getColor('grayTintLighter'))};
+  color: ${props => (props.color ? getColor(props.color) : getColor('type'))};
+  cursor: pointer;
+  transition: background-color .2s ease-in-out;
+  &:hover {
+	 background-color: ${props => (props.background ? getColor(`${props.background}Dark`) : getColor('grayTint'))};
+  }
+
+  ${(props) => {
+		if (props.size === 'small') {
+			return css`
+				font-size: ${getVariable('uiRegularSize')};
+				line-height: ${getVariable('uiRegularLineHeight')};
+				padding: calc(1/2 * ${getVariable('horizontalBase')}) 2.0rem;
+				border-radius: 1.7rem;
+			`;
+		}
+
+		if (props.size === 'tiny') {
+			return css`
+				font-size: ${getVariable('uiSmallSize')};
+				line-height: ${getVariable('uiSmallLineHeight')};
+				padding: calc(1/4 * ${getVariable('horizontalBase')}) calc(2/3 * ${getVariable('horizontalBase')});
+				border-radius: 1.7rem;
+			`;
+		}
+
+		return css`
+			font-size: inherit;
+			line-height: inherit;
+			padding: calc(1/3 * ${getVariable('verticalBase')}) calc(3/2 * ${getVariable('horizontalBase')});
+			border-radius: ${getVariable('verticalBase')};
+		 `;
+	}}
+`;
+
+const DefaultButton = ({
+	children,
+	PrefixIcon,
+	prefixIconColor,
+	SuffixIcon,
+	suffixIconColor,
+	size,
+	type,
+	theme: {
+		colors,
+	} = {},
+	...props
+}) => {
+	let iconSize = '1.8rem';
+	if (size === 'small') { iconSize = '1.4rem'; }
+	if (size === 'tiny') { iconSize = '1.2rem'; }
+
+	const iconStyles = {
+		NavigationChevronRight: {
+			height: iconSize,
```

Comment:
Did we end up supporting NavigationChevronRight (from the material-ui icons)?

Reply:
ah.  NO.  I can remove this.  Old code

Reply:
I removed the prefix and suffix icons.  With FontIcon we do not need them.